ui/src/common/TagChip.tsx

```diff
@@ -9,7 +9,16 @@ export const TagChip = ({color, label}: {label: string; color: string}) => {
         <Chip
             tabIndex={-1}
             variant="outlined"
-            style={{background: color, margin: '5px', color: textColor, cursor: 'text'}}
+            style={{
+                background: color,
+                margin: '5px',
+                color: textColor,
+                cursor: 'text',
+                minHeight: '32px',
+                height: 'auto',
```

Comment:
For what is setting the height needed, doesn't seem to change anything.

Reply:
Leftover from when I was testing. It has been removed.